cmake/pcl_find_libusb.cmake

```
@@ -0,0 +1,39 @@
+#
+#pcl_find_libusb is used due to VCPKG making impossible to use local findXX modules.
+# and VCPKG findlibusb doesn't create the libusb targets.
+
+# Find and set libusb
+find_package(libusb)
+
+if(TARGET libusb::libusb)
+  #libusb target is found by the find_package script. 
+  #VCPKG skip PCLs findlibusb and sets its own variables which is handled below.
+  return()
+endif()
+
+#Handle VCPKG definitions
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(libusb DEFAULT_MSG LIBUSB_LIBRARIES LIBUSB_INCLUDE_DIRS)
+
+mark_as_advanced(LIBUSB_INCLUDE_DIRS LIBUSB_LIBRARIES)
+
+if(libusb_FOUND)
```

VCPKG defines uppercase variable names

```suggestion
if(LIBUSB_FOUND)
```

---

VCPKG doesn't define the xx_FOUND variable it was done in line 16. However I have capitalized it to better match the variables that VCPG do set.

---

Not vcpkg, but you called `find_package_handle_standard_args(LIBUSB ..)` which defined therefore `LIBUSB_FOUND` ;-)